### PR TITLE
Initialize UI

### PR DESCRIPTION
**Pull request by SrinivasMushnoori**

### CODE REVIEW

File: quest.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import argparse
```
Comment (dgasmith) [inline quest.py:5]: Don't forget to add a line at the top of this file explaining what this is!

**Review by dgasmith — CHANGES_REQUESTED** [review]

**Comment by dgasmith** [comment]
@SrinivasMushnoori Ping me when this is done. Doesn't have to be perfect, but lets get it in.

File: parameters.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+qm:
```
Comment (dgasmith) [inline parameters.yml:1]: This should likely be moved into the `quest` folder. Maybe title it `default_parameters.yml`.

File: quest.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
+"""
```
Comment (dgasmith) [inline quest.py:1]: This needs to be moved into the `quest` folder as well. We usually want to call it something `CLI` for command line interface.

**Comment by SrinivasMushnoori** [comment]
@dgasmith Working on it.

**Pull request merged**